docs/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments.md

```diff
@@ -14,21 +15,82 @@ The C# compiler processes documentation comments in your code and formats them a
 Tags are processed on code constructs such as types and type members.
 
 > [!NOTE]
-> Documentation comments cannot be applied to a namespace.  
-  
- The compiler will process any tag that is valid XML. The following tags provide generally used functionality in user documentation.  
-  
-## Tags  
-  
-|||||  
-|---|---|---|---|
-|[\<c>](./code-inline.md)|[\<para>](./para.md)|[\<see>](./see.md)*|[\<value>](./value.md)  
-|[\<code>](./code.md)|[\<param>](./param.md)*|[\<seealso>](./seealso.md)*|  
-|[\<example>](./example.md)|[\<paramref>](./paramref.md)|[\<summary>](./summary.md)|  
-|[\<exception>](./exception.md)*|[\<permission>](./permission.md)*|[\<typeparam>](./typeparam.md)*|  
-|[\<include>](./include.md)*|[\<remarks>](./remarks.md)|[\<typeparamref>](./typeparamref.md)|  
-|[\<list>](./list.md)|[\<inheritdoc>](./inheritdoc.md)|[\<returns>](./returns.md)|
-  
+> Documentation comments cannot be applied to a namespace.
+
+The compiler will process any tag that is valid XML. The following tags provide generally used functionality in user documentation.
+
+## Tags
+
+:::row:::
```

Comment:
I'm not sure this syntax is better than Markdown syntax, as the column and row positions are still effectively hard-coded and you have to do a lot of shuffling to add items and there's a lot more syntax text to wade through compared to pipes separating columns.  And when rendered, it spaces out the columns all the way across the page although that isn't needed for this content. 
It does eliminate the heading build suggestion which may become a warning eventually, so  it has that going for it. Looks like this one already needs some reshuffling, as it has 4 columns in the first row and 3 columns thereafter.

Reply:
Yeah, I find the syntax verbose. One feature I really like about it (in instances like this) is that it's not rendering as an HTML table, but as a CSS grid.  This way it adapts to screen width (phone, tablet, etc). I do really like that feature, especially when it's not a table, but rather a desire for multi-column output.